astro/src/components/docs/release-notes/ReleaseNotesIssue.astro

```diff
@@ -1,4 +1,17 @@
 ---
+// Creates an issue entry in a release notes section, including the "Resolves GitHub issue #nnnn" bit
+//
+// issue: a comma-separated list of GitHub issue numbers
+//
+// Note that in our release notes docs, these are added to top-level markdown bullets. If you want to include multi-line text in the body, be sure to indent it so the renderer knows it's supposed to go with the bullet (you'll get an error if you don't).
```

Comment:
Seems like a lot to think about for a component that adds a link to a list. Go ahead and get these notes out but I am starting to think that this component is not a good idea after all